agents1/OfficialAgent.py

```diff
@@ -80,6 +80,8 @@ def __init__(self, slowdown, condition, name, folder, trust_mechanism):
         self.beliefs = None
         # Tick that the game starts at (loaded in _loaddTrustBelief from previous game if character already used)
         self.start_tick = 0
+        # Room has been visited once already
+        self._visitedOnce = []
```

Comment:
probably better to make it a set so that we can't have duplicates

Reply:
We don't really care much about duplicates though do we? If something is in there it's enough, doesn't matter how often. I also don't think time complexity matters much when talking about 16 rooms